contracts/erc20-bridge-sampler/contracts/src/DummyLiquidityProviderRegistry.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+
+contract DummyLiquidityProviderRegistry
+{
+    address private constant NULL_ADDRESS = address(0x0);
+
+    constructor()
+        public
+    // solhint-disable-next-line no-empty-blocks
+    {}
+
+    mapping (address => mapping (address => address)) internal _gAddressBook;
+
+    /// @dev Sets address of pool for a market given market (xAsset, yAsset).
+    /// @param takerToken First asset managed by pool.
+    /// @param makerToken Second asset managed by pool.
+    /// @param poolAddress Address of pool.
+    function setLiquidityProviderForMarket(
+        address takerToken,
```

Comment:
Nit - this naming may be confusing because either asset could be the taker/maker token on a given trade. Something like `xToken` / `yToken` or `firstToken` / `secondToken` may be better 🤷‍♂.